pyproject.toml

```diff
@@ -224,5 +224,14 @@ module = [
   "sphinx.*",
 ]
 
+[tool.pyright]
+include = [
+  "pylint",
```

Comment:
Wouldn't we also want `tests`?

Reply:
Ultimately yes but I assumed we would focus on pylint first